x-pack/plugins/ml/public/application/explorer/swimlane_data_utils.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isPopulatedObject } from '../../../common';
+import { OverallSwimlaneData, SwimlaneData } from './explorer_utils';
+
+export const isSwimlaneData = (arg: any): arg is SwimlaneData => {
```

Comment:
```suggestion
export const isSwimlaneData = (arg: unknown): arg is SwimlaneData => {
```

Reply:
I think we can move this function to `AnomalyTimelineService` as a `static` method.

Reply:
Updated here https://github.com/elastic/kibana/pull/105696/commits/63ff0b323f4494b208115653974e2a273a1874d6